Makefile

```diff
@@ -817,12 +817,11 @@ MACOSOUTDIR=out/macos
 
 .PHONY: release-only
 release-only:
-	@if [ "$(DISTTYPE)" != "nightly" ] && [ "$(DISTTYPE)" != "next-nightly" ] && \
-		`grep -q REPLACEME doc/api/*.md`; then \
+	@if [ "$(DISTTYPE)" = "release" ] && `grep -q REPLACEME doc/api/*.md`; then \
```

Comment:
Maybe also add `rc`? Since those are of supposed to be "complete"?
(But this comment is non blocking, since IMHO even the current state is much much better then the magical `perl` call)

Reply:
I intentionally left it just to "release", those are the only ones that *must* be complete. This is also a reflection of the current state in ci-release so I'd rather not bikeshed about this, that's what killed the last attempt to fix this.

Reply:
Yeah, I don't want to bikeshed eaither. IMO the PR as is is an important thing to land.
Resolving conversation.